README.md

```diff
@@ -1,2 +1,10 @@
 # react-native-oauth2
 OAuth2 Package for RN
+
+## Usage
+```javascript
+import Oauth2 from 'react-native-oauth2'
```

Comment:
OAuth2

Reply:
Last comment was updated with example usage which need to be considered for the design.

Reply:
fixed